solutions/instances/variables.tf

```diff
@@ -43,6 +43,151 @@ variable "prefix" {
   default     = null
 }
 
+##############################################################################
+# IBM Cloud Logs
+##############################################################################
+variable "cloud_logs_provision" {
+  description = "Provision a IBM Cloud Logs instance?"
+  type        = bool
+  default     = false
+}
+
+variable "cloud_logs_plan" {
+  type        = string
+  description = "The IBM Cloud Logs plan to provision. Available: standard"
```

Comment:
Please refer the description of other plan related variables and keep the description consistent.